build.gradle

```diff
@@ -40,7 +40,7 @@ ext {
     constraintlayoutVersion = '2.1.1'
     appcompatVersion = '1.4.1'
     coreVersion = '1.7.0'
-    coroutinesVersion = '1.6.0'
+    coroutinesVersion = '1.5.2'
```

Comment:
I was surprised by this, then I saw the commit message explaining why ( ❤️  ):

> This is done because otherwise some WCAndroid client tests fail with an 'advanceUntilIdle' and/or 'advanceTimeBy' unresolved reference error.

Reply:
Thanks for the review @mokagio !

> I was surprised by this, then I saw the commit message explaining why ( ❤️ ):

Yes, @0nko made this upgrade on the previous #56 PR, but as it seems it is actually [incompatible with WCAndroid](https://buildkite.com/automattic/woocommerce-android/builds/1399#5d20d78b-87e4-4e68-a27b-1532ab811456), the WCAndoird `Coroutines` version, which currently is the `1.5.2`. For some reason, the newest `1.6.0` version, comming from the library, overrides the one from the client. Then, this creates test failures as the newer version of `Coroutines` there were lots of major testing related changes that when upgrading need to be handled with care.

Reply:
BTW, I've been trying to smoke test the app locally, but I keep having Emulator issues etc. By now it's become personal, so I'll have to get to the bottom of it even if someone else reviews it before me. 

Besides, while I'm confident in going through the sample app in monkey mode and in verifying WooCommerce Android builds when using the local composite build version of this, I don't have much to add when it comes to the technicalities of the upgrade. It's still best for someone that actually knows Android to look over this.

Reply:
👋 @mokagio !

Thanks for trying to smoke test the app! 🙏 

> BTW, I've been trying to smoke test the app locally, but I keep having Emulator issues etc.

Having fun with Emulators... 🤗 What kind of issues are you getting, maybe I can try to help?

> By now it's become personal, so I'll have to get to the bottom of it even if someone else reviews it before me.

😅 Good luck, I hope you win this battle! 🤞

> Besides, while I'm confident in going through the sample app in monkey mode and in verifying WooCommerce Android builds when using the local composite build version of this, I don't have much to add when it comes to the technicalities of the upgrade.

👍 

> It's still best for someone that actually knows Android to look over this.

I hear you, maybe I can drag @0nko here to help with a quick review, as always through shameless pinging... 😅

Reply:
I've checked the code and tested the sample app. Everything is looking good 👍.

Reply:
Thank you so much for checking and testing this PR @0nko ! 🙇